index.html

```diff
@@ -13,10 +13,20 @@
 <body>
     <header class="header">
     <a href="" class="logo">My Logo</a>
+    <!Desktop version>
+    <nav class="desktop-menu">
+    <ul>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+    </ul>
+    </nav>
+    <!Desktop version end>
     <nav class="menu-img"><img src="images/Union.svg"></nav>
     </header>
     <section id="first-sec">
-        <h1>I'm Tebe <br> Glad to see <br> you!</h1>
+        <h1 class="mobile-h1">I'm Tebe <br> Glad to see <br> you!</h1>
+        <h1 class="desktop-h1">I'm Tebe <br> Glad to see you!</h1>
```

Comment:
- [ ] Great job making the desktop version however you also have to make sure that your previously finished mobile version stay's the same. Please 🙏 fix the `first-sec` of your mobile version to make it look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0) as mentioned in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements) of the project to make your portfolio look like figma design

> You need to make sure that your dekstop version looks like the one in the Figma design template that you chose.

 I suggest hiding `desktop-h1` when the screen is mobile size.


**_Figma design_** 

<img width="212" alt="Screenshot 2023-02-23 at 10 38 09 PM" src="https://user-images.githubusercontent.com/44798044/220986270-3ef6501b-a977-4ab1-b33f-0a761b488fe4.png">

**_Your design_**

<img width="357" alt="Screenshot 2023-02-23 at 9 18 18 PM" src="https://user-images.githubusercontent.com/44798044/220986100-857211af-29cc-4b04-af0d-c6a2516eea39.png">